modules/ocf_desktop/files/xsession/auto-lock

```diff
@@ -139,6 +139,9 @@ def main():
         sys.exit(1)  # Because we didn't lock/logout
     else:
         timebomb_explode()
+        # since xautolock killer will only be called while this script is running we sleep to
+        # keep it in the background
+        time.sleep(86400)
```

Comment:
Yeah, not a huge of this either. This is the case is that normally you put something like xscreensaver here, but we use our script which calls xflock4. However this method definitely does not work without this.